samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/mentionSupport.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.0",
```

Comment:
Can we use latest version?

Reply:
I have updated 1.5.
Teams platform supports v1. 5 or earlier of Adaptive Card.